comps/animation/wav2lip/requirements.txt

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
why don't we have versioning for all packages?  will new version of some packages cause issue to fixed version package?

Reply:
Added all versioning